### PR TITLE
Java cc28

### PR DESCRIPTION
**Pull request by wafman**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401Challenges/src/test/java/code401Challenges/QuickSortTest.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package code401Challenges;
+
+import com.sun.xml.internal.bind.v2.model.annotation.Quick;
```
Comment (evanslaton) [inline code401Challenges/src/test/java/code401Challenges/QuickSortTest.java]: This is preventing your code from compiling!

File: code401Challenges/src/test/java/code401Challenges/QuickSortTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class QuickSortTest {
+
+    @Test
+    public void testMergeSort_unevenSize_Positive(){
```
Comment (evanslaton) [inline code401Challenges/src/test/java/code401Challenges/QuickSortTest.java]: While reusing the merge sort tests was a great time-saver, you should have updated the name of the test functions so they don't all say merge sort
